package.json

```diff
@@ -460,4 +461,4 @@
     "node": "14.20.0",
     "yarn": "^1.21.1"
   }
-}
+}
```

Comment:
nit - this seems unnecessary?

Reply:
I didn't change anything here... I think it is due to I add one line in resolution? then it shift down one more line? loll

Reply:
It's one of those things that's really hard to read in the GitHub diff - I think it means you (likely unintentionally) removed the trailing newline at the end of the file. Not a big deal, and honestly probably something our precommit hooks should catch.